app/gui/view/graph-editor/src/component/node/input/area.rs

```diff
@@ -750,6 +887,7 @@ ensogl::define_endpoints! {
         on_port_code_update (Crumbs,ImString),
         on_background_press (),
         view_mode           (view::Mode),
+        requested_widgets   (ast::Id, ast::Id),
```

Comment:
I dont understand why "requested widgets" are represented of ast ID tuple. Docs or better name will be appreciated.